ketcher-autotests/tests/Templates/Functional-Groups/Functional-Group-Tools/functional-group-tools.spec.ts

```diff
@@ -674,4 +674,79 @@ test.describe('Templates - Functional Group Tools3', () => {
     await page.getByRole('tab', { name: 'Functional Groups' }).click();
     await pressButton(page, 'Save to SDF');
   });
+
+  test('Check aromatize/dearomatize tool on FG', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-2954
+    Description: Two FG's are added. Aromatize funcion is selected, nothing happens.
+    Dearomatize function is selected, nothing happens.
+    */
+    const clickCoordines = {
+      x1: 560,
+      y1: 360,
+      x2: 700,
+      y2: 360,
+    };
+
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.Bn, page);
+    await page.mouse.click(clickCoordines.x1, clickCoordines.y1);
+    await resetCurrentTool(page);
```

Comment:
Could you please try to move this code in separate function? Because it used 4 times here